ALI/ALI.py

```diff
@@ -64,6 +76,10 @@ def install_function():
                     extra_url = 'https://download.pytorch.org/whl/cu118' if platform.system() == "Windows" else 'https://download.pytorch.org/whl/cu113'
                     pip_install(f'{lib} --extra-index-url {extra_url}')
 
+                if lib=="numpy" :
+                  pip_uninstall('numpy')
+                  pip_install('numpy<2')
```

Comment:
`numpy` is shipped with Slicer, it should not be touched by `pip_install` and especially not `pip_uninstall`.

I searched the code `rg "['\"]numpy['\"]"` and also see usage in `AREG`. There is a usage in `ALI_IOS_utils/requirement.py` but I believe this is only run in the Conda environment. It should not be an issue.

Reply:
I removed those calls from `AREG` and `ALI`. 

Also - I don't think they did anything anyway. The old test ran `import numpy` to fetch `np.__version__`, however this loads the module and adds it to module cache. Subsequent `pip_uninstall` and `pip_install` would have no effect.